pacemaker.spec.in

```diff
@@ -403,14 +403,26 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun_with_restart pacemaker.service
 
+%pre remote
+systemctl --quiet is-active pacemaker_remote
+if [ $? -eq 0 ] ; then
+    touch %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
+    systemctl stop pacemaker_remote >/dev/null 2>&1
+else
+    rm -f %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
+fi
+
 %post remote
 %systemd_post pacemaker_remote.service
 
 %preun remote
 %systemd_preun pacemaker_remote.service
 
-%postun remote
-%systemd_postun_with_restart pacemaker_remote.service
```

Comment:
Let's use %systemd_postun here. Currently it's a no-op, but it's there in case systemd needs to add some housekeeping at some point. It's guaranteed not to try to restart the service.